website/source/docs/connect/proxies/envoy.md

```diff
@@ -292,20 +292,20 @@ defaults that are inherited by all services.
   service has the configuration option
   [`connect_timeout_ms`](/docs/agent/config-entries/service-resolver.html#connecttimeout)
   set for the `service-resolver`, that timeout value will take precedence over
-  this mesh gateway option.
+  this gateway option.
 
-- `envoy_mesh_gateway_bind_tagged_addresses` - Indicates that the mesh gateway
+- `envoy_gateway_bind_tagged_addresses` - Indicates that the gateway
```

Comment:
Should we still note the deprecated names somewhere? Maybe in each block we could say that prior to 1.8 this option was previously named X and that it is still usable with those names but is deprecated.

Reply:
Yeah. Checking in with Kaitlin about how we should handle that.

Reply:
Fixed